1.9/overview/concepts.md

```diff
@@ -348,7 +348,7 @@ A Mesos resource offer provides a set of unallocated resources (e.g. cpu, disk,
 
 ### <a name="mesos-containerizer"></a>Containerizer
```

Comment:
If you look at the list of concepts https://github.com/dcos/dcos-docs/pull/929/files#diff-2050705536f314fc0acd1315e4caaf06L267 they are all prefixed by mesos- (as they are mesos concepts).